CONTRIBUTING.md

```diff
@@ -38,52 +38,41 @@ Finally, set the upstream remote to the official Spyder repo with:
 $ git remote add upstream https://github.com/spyder-ide/spyder.git
 ```
 
-### Creating a conda environment or virtualenv
 
-If you use Anaconda you can create a conda environment with the following commands:
+### Creating an environment and installing dependencies
 
-```bash
-$ conda create -n spyder-dev python=3
-$ conda activate spyder-dev
-```
-
-You can also use `virtualenv` on Linux, but `conda` is **strongly** recommended:
+If you use a Conda-based distribution (Anaconda, etc), you can create an environment and install the necessary dependencies as follows:
 
 ```bash
-$ mkvirtualenv spyder-dev
-$ workon spyder-dev
+$ conda create -n spyder-dev -c conda-forge --file requirements/conda.txt
+$ conda activate spyder-dev
+(spyder-dev) $ python install_dev_repos.py
```

Comment:
```suggestion
```

This is no longer necessary because `bootstrap.py` install the subrepos by default.

Reply:
```suggestion
$ conda activate spyder-dev
```

You still need to activate the environment, though, for the next steps